module/Application/view/error/404.phtml

```diff
@@ -25,7 +33,7 @@ use Laminas\Mvc\Application;
             $reasonMessage = 'We cannot determine at this time why a 404 was generated.';
             break;
     }
-?>
+    ?>
```

Comment:
If you indent this line, you must do the same with the next line.

Reply:
I have now also indented the next line, because this output is located within an "if" statement and therefore has to be indented - according to phpcs